### PR TITLE
Display GAS price

### PR DESCRIPTION
**Pull request by hbibkrim**
**What current issue(s) from Trello/Github does this address?**
Don't know if one open

**What problem does this PR solve?**
Display GAS price

**How did you solve this problem?**
Using the coinmarketcap API to retrieve the GAS price

**How did you make sure your solution works?**
Imported changes from https://github.com/CityOfZion/neon-js/pull/5 

**Are there any special changes in the code that we should be aware of?**
Depends on https://github.com/CityOfZion/neon-js/pull/5

**Is there anything else we should know?**
Changes to neon-js means we could retrieve market prices in different currencies

- [ ] Unit tests written?

### CODE REVIEW

File: app/components/NetworkSwitch.js [hunk 1]
```diff
@@ -16,15 +16,11 @@ const initiateGetBalance = (dispatch, net, address) => {
   syncAvailableClaim(dispatch, net, address);
   syncBlockHeight(dispatch, net);
   return getBalance(net, address).then((resultBalance) => {
-    return getMarketPriceUSD(resultBalance.Neo).then((resultPrice) => {
-      if (resultPrice === undefined || resultPrice === null){
```
Comment (hbibkrim) [inline app/components/NetworkSwitch.js]: Moved this logic to neon-js

**Comment by canesin** [comment]
@hbibkrim are you in our slack ? Ping me there ...  [if not join us](https://join.slack.com/t/neoblockchainteam/shared_invite/MjE3ODMxNDUzMDE1LTE1MDA4OTY3NDQtNTMwM2MyMTc2NA).

**Comment by Ejhfast** [comment]
@canesin yes @hbibkrim is! we have been talking

**Comment by hbibkrim** [comment]
I got kinda excited and implemented the functionality to view prices (#44) between currencies as well. Whenever the settings page gets done, we could set the default currency there?

**Comment by dop89** [comment]
@hbibkrim I have an approach for currency switching. Please have a look here: https://github.com/CityOfZion/neon-wallet/pull/51

Unfortunately I could not resolve the conflicts yet. I try to do it this weekend. Settings page for default currency would be pretty nice. 

I think we should wait until this PR (https://github.com/CityOfZion/neon-js/pull/7) is merged into master. Then we have a general `getMarketPrice` and `getMarketPrices` function.

**Comment by hbibkrim** [comment]
Closing PR as API used to get price has been reverted to bittrex from coinmarketcap as coinmarketcap was unreliable

**Pull request closed without merging**